OP2Utility.vcxproj.filters

```diff
@@ -177,9 +177,12 @@
     <ClInclude Include="src\Map\SavedGameUnits.h">
       <Filter>Map</Filter>
     </ClInclude>
+    <ClInclude Include="src\Stream\SliceReader.h">
+      <Filter>Stream</Filter>
+    </ClInclude>
     <ClInclude Include="src\Tag.h">
       <Filter>Header Files</Filter>
-    </ClInclude>
+	</ClInclude>
```

Comment:
Hmm, looks like there is a tab/space inconsistency here. I wonder how that got introduced.

Looking into this, I found another inconsistency in the Test project file filters as well.

I'll update.